HospitalSystem.ConsoleUI/Director/RenovationUI.cs

```diff
@@ -0,0 +1,83 @@
+using HospitalSystem.Core;
+
+namespace HospitalSystem.ConsoleUI;
+
+public class RenovationUI : RoomUI
+{
+    public RenovationUI(Hospital hospital, List<Room> loadedRooms) : base(hospital, loadedRooms)
+    {
+
+    }
+
+    public override void Start()
+    {
+        throw new NotImplementedException("Whoops");  // TODO: don't have such an inheritance, it bad
+    }
```

Comment:
Shouldn't this be implemented?

Reply:
Nah, won't hook into this one in this way, only using its public funcs. Will do in a better way in future